src/views/issuance/components/zapV2/constants/index.ts

```diff
@@ -0,0 +1,99 @@
+import { ChainId } from 'utils/chains'
+import { Address } from 'viem'
+
+const mainnetTokens = [
+  {
+    address: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE' as Address,
```

Comment:
define array type to avoid Address casting on every item, `mainnetTokens: ZapToken[] = [...]`